AppCenterReactNativeShared/Products/local.podspec

```diff
@@ -0,0 +1,18 @@
+# This podspec is intended to be used as local pod reference during development,
+# while the `AppCenterReactNativeShared.podspec` is only used in the official SDK releases.
+
+Pod::Spec.new do |s|
+  s.name              = 'AppCenterReactNativeShared'
+  s.version           = '2.1.0'
+  s.summary           = 'React Native plugin for Visual Studio App Center'
+  s.license           = { :type => 'MIT',  :file => 'AppCenterReactNativeShared/LICENSE' }
+  s.homepage          = 'https://github.com/microsoft/appcenter-sdk-react-native'
+  s.documentation_url = "https://docs.microsoft.com/en-us/appcenter/"
+  s.author            = { 'Microsoft' => 'appcentersdk@microsoft.com' }
+  system("SRCROOT=#{__dir__}/../ios #{__dir__}/../prepare-local-podspec.sh")
```

Comment:
What is this one?

Reply:
In order for `pod install` to work, we need to generate `AppCenter-SDK-ReactNative-iOS-Pod-{version}.zip` and this `system` call is to execute `prepare-local-podspec.sh` shell script to do so.